chrome/content/zotero/containers/itemTree.jsx

```diff
@@ -0,0 +1,3430 @@
+/*
+	***** BEGIN LICENSE BLOCK *****
+	
+	Copyright © 2019 Center for History and New Media
+					George Mason University, Fairfax, Virginia, USA
+					http://zotero.org
```

Comment:
Should update all of these new files to say CDS instead:

https://github.com/zotero/zotero/blob/328865299b9f87eee7b40ee944f720b09fe0a089/chrome/content/zotero/components/search.jsx#L4-L6

(Maybe worth squashing those, if it doesn't totally blow up the merge commit.)